BuildScript/BuildTargets/Testing.targets

```diff
@@ -48,7 +48,11 @@
 
     <Error
       Text="Running Unit-Tests failed because of $(_failures) failures in: %0A@(FailedProjects->'%(Identity) (%(Platform)/%(DatabaseSystem)) with %(Failures) failures','%0A')"
-      Condition="$(_failures) &gt; 0"/>
+      Condition="$(_failures) &gt; 0 and '$(_nunitIsSuccess)' == 'False'"/>
```

Comment:
Do we need this second check here? if there's failures, we don't care about nunitisuccess.